bip-0345.mediawiki

```diff
@@ -5,7 +5,7 @@
   Author: James O'Beirne <vaults@au92.org>
           Greg Sanders <gsanders87@gmail.com>
   Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0345
-  Status: Draft
+  Status: Withdrawn
```

Comment:
The README would need updating as well.

Reply:
Probably a diff like this

```diff
diff --git a/README.mediawiki b/README.mediawiki
index fe3be265..2ea96ed3 100644
--- a/README.mediawiki
+++ b/README.mediawiki
@@ -1127,13 +1127,13 @@ Standard
 | Final
-|-
+|- style="background-color: #ffcfcf"
 | [[bip-0345.mediawiki|345]]
 | Consensus (soft fork)
 | OP_VAULT
 | James O'Beirne, Greg Sanders
 | Standard
-| Draft
+| Withdrawn
 |-
 | [[bip-0347.mediawiki|347]]
 | Consensus (soft fork)
```

Reply:
Based on your comment, you may also want to add `Superseded-By: [bip-number-of-OP_CCV-coming-soon™]` below the Post-History header.

Reply:
I.e., `Superseded-By: 443`.